lib/UnoCore/Targets/Android/build.sh

```diff
@@ -22,8 +22,14 @@ exit 1
 export JAVA_HOME="@(JDK.Directory)"
 #endif
 
-./gradlew @(Gradle.Task) "$@"
+./gradlew @(APK.Gradle.Task) "$@"
```

Comment:
We also need similar changes in `build.bat` for Windows.

Reply:
Omg! I always forgot Windows users, I am sorry. fix coming right way